UI/app_update.ts

```diff
@@ -364,6 +366,40 @@ export async function* UI_Interaction_Update(
                 }
             }
             model.rightPanelModel.show = true;
+        } else if (event.type == "ViewNoteDetail") {
+            let root: NostrEvent;
+            root = event.event;
+            if (event.event.parsedTags.root) {
+                const res = model.app.eventSyncer.syncEvent(NoteID.FromHex(event.event.parsedTags.root[0]));
+                if (res instanceof Promise) {
+                    continue;
+                }
```

Comment:
why `continue`?

Reply:
Because if the no root event we can sync, I want to stop the ui render at this time